### DISCUSSION TITLE
Interest in 2D visualization for small particles

### DISCUSSION THREAD

**Discussion by rubenarciba** [MDAnalysis/mdanalysis#4470] [GSoC Discussions]
Hi, I am Ruben Arciba. I am a 2nd year Computer Science student. I know Python and Java. I look forward to work on this project and get hands-on experience

**Comment by jennaswa** [comment]
Welcome to MDAnalysis, Ruben! Please see our recent [GSoC blog post](https://www.mdanalysis.org/2024/02/27/gsoc2024/) to learn more about how to get started and how to apply to work with MDAnalysis.
